src/main/groovy/edu/oregonstate/mist/locations/frontend/resources/LocationResource.groovy

```diff
@@ -73,35 +59,25 @@ class LocationResource extends Resource {
         this.endpointUri = endpointUri
     }
 
-    @Context
-    UriInfo uriInfo
-
     @GET
-    @Produces(MediaType.APPLICATION_JSON)
     @Timed
     Response list(@QueryParam('q') String q,
                   @QueryParam('campus') String campus, @QueryParam('type') String type,
```

Comment:
What if I want to query multiple types? I think we talked about doing something like type=blah&type=building which would be handled as an array of strings. Could you also update the swagger spec to indicate a collectionFormat for the type parameter?